core/src/test/scala/unit/kafka/server/MetadataCacheTest.scala

```diff
@@ -187,49 +191,24 @@ class MetadataCacheTest {
       new UpdateMetadataBroker()
         .setId(1)
         .setEndpoints(broker1Endpoints.asJava))
-    verifyTopicMetadataPartitionLeaderOrEndpointNotAvailable(brokers, sslListenerName,
+    val metadataCacheBrokerId = 9
```

Comment:
Would 0 work just as well? It would make the test more coherent if we used one of the ids included in the UpdateMetadata request.

Reply:
Done